### ISSUE TITLE
Incorrect rates for 1kk+

### ISSUE THREAD

**Issue by EIZENHORN91** [skidvis/unity-fee-calculator#1]
![image](https://github.com/skidvis/unity-fee-calculator/assets/25122510/45e011a9-094e-4719-aa43-d13d3769f9ea)
![image](https://github.com/skidvis/unity-fee-calculator/assets/25122510/f0a12c81-2724-4ab8-80ee-275620ce60f9)

**Comment by thygrrr** [comment]
Yes, I got a huge "bill" from Unity and wondered why the 3rd million of installs is so expensive.
(the values are wrong)
